cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_service_manager.go

```diff
@@ -559,10 +588,11 @@ func (csm *cloudServiceManager) getAsgTemplate(groupID string) (*asgTemplate, er
 				vcpu: vcpus,
 				ram:  int64(flavor.Ram),
 				zone: az,
+				tags: tags,
 			}, nil
 		}
 	}
-	return nil, nil
+	return nil, fmt.Errorf("no related instance flavor:%s found in as group:%s", *configuration.InstanceConfig.FlavorRef, groupID)
```

Comment:
This error might be not precise enough. In what situation that we can't build a node template?

Reply:
In one as group, all az sold out this instance flavor(I tested, if sold out, the az will not give this flavor info)